src/app/modules/submission/email-submission/email-submission.controller.ts

```diff
@@ -317,6 +321,22 @@ const submitEmailModeForm: ControllerHandler<
             meta: logMetaWithSubmission,
           })
 
+          // TODO 6395 make responseMetadata mandatory
+          if (responseMetadata)
+            statsdClient.distribution(
+              'formsg.submissions.normResponseTimeMetadata',
+              getNormalisedResponseTime(
+                responseMetadata.responseTimeMs,
+                responseMetadata.numVisibleFields,
+              ),
+              1,
+              {
+                mode: 'email',
+                responseTimeMs: '${responseMetadata.responseTimeMs}',
+                numOfVisibleFields: '${responseMetadata.numVisibleFields}',
```

Comment:
⚠️⚠️ We cannot release this! Tag cardinality on DataDog metrics should stay low (less than 1000 per metric if possible). Tracking the responseTime in ms will explode cardinality and explode our bill.

Even `numOfVisibleFields` probably shouldn't be added as a tag given we know there are broad variety of form sizes, up to thousands of fields for crazy forms.